MAINTAINERS.md

```diff
@@ -0,0 +1,144 @@
+# Making a Release
+
+* Create a release branch.
+
+  ```sh
+  now=`date -u +%Y%m%dT%H%M%S`
```

Comment:
What do you think about ISO date strings, which are much more readable

```suggestion
  now=`date -u +%Y-%m-%dT%H:%M:%SZ`
```

though I could do without the Z.

Reply:
I could do without the time entirely. Posted a change that reflects what I did instead for this step, as can be seen in the PR branch name.